DEPLOYMENT.md

```diff
@@ -20,6 +20,9 @@ The server requires:
 - Accessible PostgreSQL endpoint.
 - Exposing the configured HTTP port outside.
 
+Optionally:
```

Comment:
Let's have it this way then:

The server requires:

- (optional) `SENTRY_KEY` and `SENTRY_PROJECT` environment variables to enable error logging.
- (optional) `AUTH0_DOMAIN` and `AUTH0_AUDIENCE` environment variables to enable authorization.
- Accessible PostgreSQL endpoint with the metadata.
- Accessible PostgreSQL endpoint with the server state.
- Exposing the configured HTTP port outside.